test/ApiSpec.scala

```diff
@@ -121,11 +128,21 @@ class ApiSpec
         createdAtom.id mustEqual "2"
       }
     }
-    "call out to publisher to publish an atom" in withApi() { api =>
+    "call out to live publisher to publish an atom" in withApi() { api =>
       val result = call(api.publishAtom("1"), requestWithCookies(api))
       status(result) mustEqual NO_CONTENT
     }
-    "call report failure if publisher fails" in withApi(publisher = failingMockPublisher) { api =>
+
+    "call out to preview publisher when adding an asset" in withApi() { api =>
```

Comment:
Let's add a test here to catch the situation mentioned in the above comment: so we can use the mock publisher `verify()` to confirm that the atom update includes the new asset.